visual-js/visual-snapshots/src/commands/pdf.ts

```diff
@@ -28,11 +28,16 @@ export const snapshotNameOption = new Option(
     " Supports the following parameters: {filename}, {page}"
 );
 
+export const recursiveOption = new Option(
+  "--recursive",
+  "If specified, will search recursively for PDF files in passed directories."
+).default(false);
+
 export const pdfCommand = (clientVersion: string) => {
   return new Command()
     .name("pdf")
     .description("Create visual snapshots for each page of a PDF file")
-    .argument("<pdf-file-path>", "A path to a PDF file")
+    .argument("<globs or dirs...>", "A path to a PDF file")
```

Comment:
We should update argument description and point out it accepts: 
- a path to a pdf file, directory with pdf files, or a glob pattern.

Reply:
Updated